app/src/main/res/layout/onboarding_slide_final.xml

```diff
@@ -45,7 +45,7 @@
         android:layout_marginBottom="16dp"
         android:fontFamily="sans-serif-medium"
         android:gravity="center_horizontal"
-        android:maxLines="1"
+        android:maxLines="2"
```

Comment:
I think we should remove this `maxLines` attributes completely for this screen. So that even if in some screen it goes to three-lines the UI will be visible fully.

Reply:
Changes Applied.